airwallex/src/main/java/com/airwallex/android/view/PaymentMethodsViewModel.kt

```diff
@@ -96,67 +93,69 @@ internal class PaymentMethodsViewModel(
         return resultData
     }
 
-    private fun retrieveAvailablePaymentMethods(
+    private suspend fun retrieveAvailablePaymentMethods(
         availablePaymentMethodList: MutableList<AvailablePaymentMethodType> = mutableListOf(),
         availablePaymentMethodPageNum: AtomicInteger = AtomicInteger(0),
         resultData: MutableLiveData<Result<List<AvailablePaymentMethodType>>>,
         clientSecret: String
     ) {
-        airwallex.retrieveAvailablePaymentMethods(
-            params = RetrieveAvailablePaymentMethodParams.Builder(
-                clientSecret = clientSecret,
-                pageNum = availablePaymentMethodPageNum.get()
+        val response = try {
+            airwallex.retrieveAvailablePaymentMethods(
+                session = session,
+                params = RetrieveAvailablePaymentMethodParams.Builder(
+                    clientSecret = clientSecret,
+                    pageNum = availablePaymentMethodPageNum.get()
+                )
+                    .setActive(true)
```

Comment:
indentation looks a little off here

Reply:
Umm..I used Reformat Code function (option+command+L) in android studio, even if I removed 4 spaces, it will reformat as it is.

Reply:
And actually, this indentation is what detekt expects, otherwise it complained after I removed some extra spaces